src/project/qml/MuseScore/Project/internal/NewScore/KeySignatureListView.qml

```diff
@@ -76,7 +76,13 @@ GridView {
             navigation.panel: root.navigationPanel
             navigation.row: root.columns === 0 ? 0 : Math.floor(model.index / root.columns)
             navigation.column: model.index - (navigation.row * root.columns)
-            navigation.accessible.name: keySignature.text
+            navigation.accessible.name: {
+                if (isSelected) {
+                    return keySignature.text;
+                } else {
+                    return keySignature.text + qsTr("Not Selected");
```

Comment:
Wouldn't we need a space between the key signature text and "Not selected"?

Also, we prefer our own `qsTrc` for translating, which would be used like this:
```suggestion
                    return keySignature.text + " " + qsTrc("project/newscore", "Not selected");
```
("project/newscore" is the translation context here.)

And finally, we prefer to use "Sentence case" instead of "Title Case", i.e. start `selected` with a lowercase `s`. (Of course, screen reader users won't notice the difference, but let's keep it consistent anyway 🙂)

Reply:
Sure, Thanks for guiding me. 
@cbjeukendrup Updated .